src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspace.cs

```diff
@@ -23,6 +24,10 @@ public abstract class VisualStudioWorkspace : Workspace
         internal VisualStudioWorkspace(HostServices hostServices)
             : base(hostServices, WorkspaceKind.Host)
         {
+            // Compute the size of a solution in the bg in vs workspaces.  This will ensure that we
+            // create a persistence service in-proc for VS if the solution size warrants it.
+            Options = Options.WithChangedOption(SolutionSizeOptions.ComputeSolutionSize, true);
```

Comment:
@Therzok this is what you would need in vs4mac.

Reply:
Perfect, thanks!

Reply:
What's "bg"?

Reply:
background.  :)

Reply:
(updated comment)